TaskList/TaskList/TaskListHelper.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TaskList
+{
+    internal class TaskListHelper
+    {
+        internal static int count = 0;
+
+        private readonly TaskList[] taskList = new TaskList[20];
+
+        public void FilterBy()
+        {
+            Console.WriteLine($"Input how u want sort \n1. By Priority \n2. By Date");
+
+            switch (Program.ParseInt(Console.ReadLine()))
+            {
+                case 1:
+                    FilterPlansByPriority();
+                    break;
+                case 2:
+                    FilterPlansByDate();
+                    break;
+            }
+        }
+
+        public void EditPlans()
+        {
+            Console.WriteLine("Input task number which u want edit");
+            int inputId = Program.ParseInt(Console.ReadLine());
+
+            if (inputId > count)
+            {
+                Console.WriteLine("Error. number task is no found");
+                return;
+            }
+
+            Console.WriteLine("Edit task:");
+            var inputTaskDate = Console.ReadLine();
+            var task = ParseNewTask(inputTaskDate);
+            taskList[inputId - 1] = task;
+        }
+
+        public void SavePlansTxt()
+        {
+            File.Delete("task.txt");
+            for (int i = 0; i < count; i++)
+            {
+                TaskList task = taskList[i];
+                RegularTask taskDate = new RegularTask();
+
+                try
+                {
+                    using (StreamWriter writeText = new StreamWriter("task.txt", true, System.Text.Encoding.Default))
+                    {
+                        writeText.WriteLine(task?.TaskConvert(i));
```

Comment:
we don't expect this task to be null, huh?
```suggestion
                        writeText.WriteLine(task.TaskConvert(i));
```